src/stylesheets/application.scss

```diff
@@ -0,0 +1,9 @@
+@import 'bootstrap_manifest';
```

Comment:
@rafayet-monon As the `bootstrap` is the third-party, should we move this to `/vendor` folder?
![Screen Shot 2563-08-27 at 12 29 31](https://user-images.githubusercontent.com/749672/91388161-f7326180-e860-11ea-9063-43cfcefbd441.png)

Reply:
As I understood the `vendor` is to override the third party styles. But as we are using bootstrap as a framework just like -
![image](https://user-images.githubusercontent.com/14927672/91394282-58a40180-e85c-11ea-8ab3-68bd4d7794bd.png)

shouldn't we import it at top? I thought of `bootstrap_manifest.scss` as a manifest like `application.scss` to import modules so put it at the root. What do you think?